diff --git a/vendor/pantheon-systems/upstream-management/src/Command/UpstreamRequireCommand.php b/vendor/pantheon-systems/upstream-management/src/Command/UpstreamRequireCommand.php
--- a/vendor/pantheon-systems/upstream-management/src/Command/UpstreamRequireCommand.php
+++ b/vendor/pantheon-systems/upstream-management/src/Command/UpstreamRequireCommand.php
@@ -18,7 +18,7 @@ class UpstreamRequireCommand extends RequireCommand
     /**
      * {@inheritdoc}
      */
-    protected function configure()
+    protected function configure(): void
     {
         parent::configure();
         $this
